hellodagger/src/main/java/com/airbnb/mvrx/helloDagger/HelloFragment.kt

```diff
@@ -0,0 +1,31 @@
+package com.airbnb.mvrx.helloDagger
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import com.airbnb.mvrx.*
+import kotlinx.android.synthetic.main.fragment_hello.*
+
+class HelloFragment: BaseMvRxFragment() {
+
+    private val viewModel: HelloViewModel by fragmentViewModel()
+
+    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View? {
+        return inflater.inflate(R.layout.fragment_hello, container, false)
```

Comment:
You can put this layout in the constructor of BaseMvRxFragment now :)

Reply:
Thanks, good idea!